src/Modal/__tests__/Modal.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {render} from '@testing-library/react';
```

Comment:
Prettier doesn't seem to be adding spaces around the imports inside the `{ render }`